projects/github-actions/repo-gardening/src/utils/get-next-valid-milestone.js

```diff
@@ -49,7 +49,7 @@ async function getOpenMilestones( octokit, owner, repo ) {
  */
 async function getNextValidMilestone( octokit, owner, repo, plugin = 'jetpack' ) {
 	// Find all valid milestones for the specified plugin.
-	const reg = new RegExp( '^' + plugin + '\\/\\d+\\.\\d' );
+	const reg = new RegExp( '^' + plugin + '\\/(\\d+\\.\\d|next)' );
```

Comment:
At this point I think it would be worth expanding the comment a bit to explain what is a valid milestone, what formats we can expect.